apps/rpc/src/modules/event/event-repository.ts

```diff
@@ -133,6 +133,17 @@ export function getEventRepository(): EventRepository {
                   }
                 : undefined,
             },
+            {
```

Comment:
monster query brewing (not a problem, we moght need indexes though, problem for later 🍺)